app/views/inbox/show.html.erb

```diff
@@ -16,7 +16,7 @@
 
   <%= will_paginate @inbox_comments %>
 
-  <%= form_tag user_inbox_path(@user, page: params[:page], filters: params[:filters]), method: :put, id: 'inbox-form' do %>
+  <%= form_tag user_inbox_path(@user, page: params[:page], filters: params[:filters]), method: :put, id: 'inbox-form', class: 'inbox manage' do %>
```

Comment:
If I'm going to have to adjust the CSS for the inbox, then, damn it, I am going to do it with _class_ selectors